functests/4_latency/latency.go

```diff
@@ -413,3 +420,15 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 
 	return latencies[1]
 }
+
+func removeLogfile(node *corev1.Node) {
+	cmd := []string{"cat", fmt.Sprintf("/rootfs/var/log/%s.log", logName)}
+	out, err := nodes.ExecCommandOnNode(cmd, node)
+	Expect(err).ToNot(HaveOccurred())
+	fmt.Printf("the test's output log is: \n %s \n", out)
+
+	cmd = []string{"rm", "-fr", fmt.Sprintf("/rootfs/var/log/%s.log", logName)}
```

Comment:
let's not use `-r` here please unless it is really needed (why?)

Reply:
it is for recursive deleting, & we actually don't need it indeed, I removed it . Thanks!